targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F429xI/TARGET_NUCLEO_F429ZI/PeripheralNames.h

```diff
@@ -57,6 +57,7 @@ typedef enum {
     UART_8 = (int)UART8_BASE
 } UARTName;
 
+#define SPI_COUNT 6
```

Comment:
I think this addition is not needed for this PR ?

Reply:
It wasn't until yesterday.
https://github.com/ARMmbed/mbed-os/pull/9469 introduced the static array (it was put back yesterday), so it is needed.

Reply:
#9469 is not merged.

Reply:
No it is not, but as said in the description, this PR depends on https://github.com/ARMmbed/mbed-os/pull/9469

Reply:
Then all the STM targets need to be updated, why updating F429ZI only ?

Reply:
Background is stated on the description. Other STM targets would work like they would before like stated in the https://github.com/ARMmbed/mbed-os/pull/9469

>No, it means that targets that don't provide this extra facility retain the existing single whole-system SPI mutex.

Reply:
But why would this change be valid for WISUN and F429 only ?  Does the change have drawbacks that we don't want to deploy everywhere ?  If there are only advantages and no drawbacks, I'd prefer to have the change deployed widely. We try to keep all STM targets behavior as aligned as possible otherwise maintenance becomes more complex.

Reply:
AFAIK there are mostly advantages, bit larger memory footprint because of the static array for peripherals is a drawback though. The changes done now are preceding a major change to SPI that are coming in a future release. (That will effectively overwrite the changes done now)

Wi-SUN testing has been mainly done with F429, so that is why it is the first one to get changes. Changes to different HAL's can be added later on if there is a need.

Reply:
ok then, but when the "major changes" are made, all targets should be aligned.